### PR TITLE
keykeeper: do not schedule key add and reveal based on block

### PR DESCRIPTION
**Pull request by p4u**
Simplify keykeeper approach and do not use block to schedule actions (since now elections are based on timestamps).

Remove all storage layer wich is legacy and no longer necessary. Do not schedule, just send transactions when the event of new process or process status change, is executed.

Modify docker suit so it waits at least until height 8 to start. Else the keykeeper might not be yet ready.

### CODE REVIEW

File: vochain/appsetup.go [hunk 1]
```diff
@@ -61,7 +66,9 @@ func (app *BaseApplication) SetDefaultMethods() {
 	})
 	app.SetFnMempoolPrune(app.fnMempoolRemoveTxTendermint)
 	app.SetFnSendTx(func(tx []byte) (*cometcoretypes.ResultBroadcastTx, error) {
-		result, err := app.NodeClient.BroadcastTxSync(context.Background(), tx)
+		ctx, cancel := context.WithTimeout(context.Background(), transactionSendTimeout)
+		defer cancel()
+		result, err := app.NodeClient.BroadcastTxSync(ctx, tx)
```
Comment (altergui) [inline vochain/appsetup.go:71]: why is this change included in this commit? please split the changes of `vochain/appsetup.go` into a different commit, or if it's needed somehow for the keykeeper refactor, then add a comment maybe to transactionSendTimeout const? or at least to the commit message

File: dockerfiles/testsuite/env.miner2 [hunk 2]
```diff
@@ -1,6 +1,6 @@
 VOCDONI_DATADIR=/app/run
 VOCDONI_MODE=miner
-VOCDONI_LOGLEVEL=info
+VOCDONI_LOGLEVEL=debug
```
Comment (altergui) [inline dockerfiles/testsuite/env.miner2]: i'd leave both miners in info loglevel, 
else this clutters the CI log, when there's a failure

**Review by altergui — APPROVED** [review]
love the simplification <3

in general LGTM after reading the diff. didn't actually test it but YOLO :rocket:

File: vochain/appsetup.go (hunk 1 above)
Comment (p4u) [reply vochain/appsetup.go:71]: I have been stock on a deadlock for a while... Then I realized we do not have context on sendTx, which might be an issue. So I just added it.
